src/devices/SensorHub/SensorHub.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net5.0;netcoreapp2.1</TargetFrameworks>
```

Comment:
Please rebase and update: This line should be changed to `<TargetFrameworks>$(DefaultBindingTfms)</TargetFrameworks>`, since we now also support netstandard2.0